src/styles/card.scss

```diff
@@ -0,0 +1,90 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid $border-white;
+
+  &__container {
+    display: flex;
+    flex-direction: column;
+    padding: 32px 16px 16px;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    align-self: center;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    margin: 0 0 4px;
+    color: $main-black;
+  }
+
+  &__code {
+    line-height: 14px;
+    align-self: start;
+    margin: 0 0 16px;
+    color: $secondary-grey;
+  }
+
+  &__marks {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    line-height: 14px;
+    text-align: center;
+    margin: 0;
+    color: $main-black;
+  }
+
+  &__price {
+    width: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 16px;
+
+    &-title {
+      font-size: 12px;
+      margin: 0;
+      color: $secondary-grey;
+    }
+
+    &-count {
+      font-weight: 700;
+      font-size: 16px;
+      margin: 0;
+      color: $main-black;
+    }
+  }
+
+  &__btn {
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-transform: uppercase;
+    text-decoration: none;
+    width: 27px;
```

Comment:
why width is `27px`? is should be full width (`100%`)

Reply:
But with this decesion I can't pass tests, thats why I decided make width: 27px and use paddings instead of width 100%
![3](https://github.com/mate-academy/layout_product-cards/assets/91736996/1a19b7b7-44e0-4036-aa2b-30c06e7e20f2)

Reply:
Fixed this problem by using box-sizing: border-box; passed all tests! Thank you!